src/lib/ens.ts

```diff
@@ -0,0 +1,40 @@
+// @ts-ignore TODO: add type declarations for ensjs module
```

Comment:
```suggestion
```

please fix in https://github.com/shapeshift/web/blob/8036760e16e69aaf1b2eb6f8387940b7972cf7e4/src/react-app-env.d.ts

Reply:
dang, how'd i miss missing types?!? what am I even doing with my life?